aptos-move/framework/aptos-framework/sources/account.move

```diff
@@ -86,7 +86,7 @@ module aptos_framework::account {
     native fun create_address(bytes: vector<u8>): address;
     native fun create_signer(addr: address): signer;
 
-    public fun initialize(account: &signer,
+    public(friend) fun initialize(account: &signer,
```

Comment:
Make all initialize functions friend only so they can't be called outside of genesis, for example in governance proposals.